extendr-api/src/graphics/mod.rs

```diff
@@ -544,10 +651,10 @@ impl Device {
     /// Draw a stroked/filled axis-aligned rectangle.
     /// gc.color() is the stroke color.
     /// gc.fill() is the fill color.
-    pub fn rectangle(&self, from: (f64, f64), to: (f64, f64), gc: &Context) {
+    pub fn rect(&self, from: (f64, f64), to: (f64, f64), gc: &Context) {
```

Comment:
I want to choose `rect()` here. I believe "rect" is more popular than "rectangle" in 2D drawing libraries.

* https://processing.org/reference/rect_.html
* https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect
* https://docs.rs/lyon/latest/lyon/math/fn.rect.html